src/collections/_documentation/platforms/javascript/default-integrations.md

```diff
@@ -49,13 +49,13 @@ Available options:
 
 ```js
 {
-  beacon: boolean;
-  console: boolean;
-  dom: boolean;
-  fetch: boolean;
-  history: boolean;
-  sentry: boolean;
-  xhr: boolean;
+  beacon: boolean;  // Log HTTP requests done with the Beacon API
+  console: boolean; // Log calls to `console.log`, `console.debug`, etc
+  dom: boolean;     // Log click and keypress events that bubbled up to `document`
```

Comment:
It captures all user interactions, there's a mistake in code comment.

Reply:
Do you mean it also captures DOM events that were already handled? This came up during a support ticket where they wanted to know more details than "all user interaction"